### DISCUSSION TITLE
Can PnP Results Page be Dynamically Linked to SP List Web Part

### DISCUSSION THREAD

**Discussion by RamBalani** [microsoft-search/pnp-modern-search#2009] [Q&A] [answered]
We have implemented a Pnp Search box paired with a PnP Search Results web part BOTH on the same Modern Page.
We need to add another customary SP List web part and have this NEW List DYNAMICALLY FILTERED such that when a List row is SELECTED on the PnP results page above it , the NEW bottom most  list web part is dynamically filtered by a common column shared in both results page (after a search!) and the new List web part (* we have configured PnP Results page to allow selection on the results page)

**Comment by erikthomte** [comment, marked as answer]
Short answer: No

Take a look at Franky's: blogpost https://franckcornu.com/post/modern-search-focus-item-selection/
Quote: "_OOTB SharePoint Web Parts can only be used as source, not targets._"

**Reply by RamBalani** [reply]
**### RESOLVED!** Mapped in Schema RefinableString02 but (dummy) used RefinableString04 on the target result web part ( * oy veh !*) ### 
--------------------------------
I followed this - t https://franckcornu.com/post/modern-search-focus-item-selection/  as suggested--and created two PnP Results web part with two different list - i.e Source List on Top - Target on the bottom screen.
Can't seem to get it to work, i.e. when selecting 7.3.9 (see attachments) -that selected value is not filtering the bottom PnP Result page web part.
See these attachments:
1- Shows Top Results Web part with item 7.3.9 value
![1-Top List Shows 7-3-9 List Row  Items](https://user-images.githubusercontent.com/37209670/162582131-f85d9c1c-94f7-4194-be87-18ee86b56696.jpg)

===========
2 Bottom Result Web part shows also item with 7.3.9 - i.e. very LAST ROW!
![2-Bottom List Shows 7-3-9 Also-Last Row](https://user-images.githubusercontent.com/37209670/162582166-c4e5f4e1-0afc-4648-b877-f74989642824.jpg)

===========
3 Top List item/row is SELECTED
![3-Top List Select Works 7-3-9 is Selected Shown](https://user-images.githubusercontent.com/37209670/162582187-cd4f2e69-5ed3-44c1-9e11-7e78f6f8b0e0.jpg)

============
4 RefianableString02 is defined in Schema to include both crawled column properties from these two list 
![4-RefinaleString02 Defined in Schema](https://user-images.githubusercontent.com/37209670/162582255-0e7fad9d-c3b0-412d-8b22-f14da8e7d28c.jpg)

================
5 Pnp TARGET Results Page setting per Cornu documentation 
![5-PnP Search Results Setting for Connecting to Result Source](https://user-images.githubusercontent.com/37209670/162582293-a8513924-4853-44fa-b23a-c3d15c16a75a.jpg)

===========
6 PnP Source (TOP) List Result Page Setting 
![6-Top List-Source Result Web Part Setting](https://user-images.githubusercontent.com/37209670/162582545-ef4f1359-56af-4741-ad9a-597d61fbaca7.jpg)

===================
7 No selected value filtering-i.e. no records found -the filtering seems to MISFIRE! WHY? WHAT AM I DOING WRONG?
![7-Bottom List to Filter Finds Nothing-ie value of Select not Filtering](https://user-images.githubusercontent.com/37209670/162582599-6b8e8d24-a1ee-4640-857a-1f586b9bee6b.jpg)

=========================

**Comment by RamBalani** [comment]
Thanks for the reply and reference both.

 I take it that this statement "OOTB SharePoint Web Parts can only be used as source, not targets." is meant to READ as below----
 "OOTB SharePoint Web Parts can only be used as source, not targets **since one of the   List web parts is  'contained' in the PnP results page " - right ?** 

Outside of PnP results List  -an OOTB Sharepoint list web part can be either SOURCE or TARGET (nothing inherent prevents that) as desired for a specific purpose.  
Seems like this would be a hugely desirable feature for PnP search to have, i.e. make the PnP results list web part  be the SOURCE NOT the TARGET. (i.e. OUTBOUND)!

 In our case - we are allowing ISO-13485 medical device quality standard to be search in the PnP results-then when user selects a standard clause-- the native List OOTB web part below the Pnp results  should ONLY present Document Toolkits being sold that addresses the PnP results standard row selected-not the other way around, by way of an example. That defeats our purpose.

**Comment by wobba** [comment]
See official documentation at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/connections/item-selection/

**Reply by RamBalani** [reply]
Seen this -in fact was reviewing earlier.
Seems that what I ask for is doable if BOTH LISTS ARE CONTAINED INSIDE A PnP RESULTS page independently BUT not when only when one list is PnP and other OOTB native stand alone list embedded into the SAME MODERN PAGE.
Am testing that one ! Thanks.
